backend/index.js

```diff
@@ -18,6 +23,13 @@ client.on('error', err => {
 app.use(cors());
 app.use(express.json());
 app.use(express.urlencoded({ extended: true }));
+app.use(cookieParser())
+app.use(function(req, res, next) {
+  res.header("Access-Control-Allow-Origin", "*");
+  res.header("Access-Control-Allow-Methods", "GET, PUT, POST");
+  res.header("Access-Control-Allow-Headers", "Origin, X-Requested-With, Content-Type, Accept");
+  next();
+});
 
 let roomStore = [];
 
```

Comment:
- [ ] Perhaps we can implement a second array to keep track of logged in admins here

Reply:
yes we should do that in a later PR.